modules/authentication/proc-creating-a-custom-transformer-to-provision-users-from-rhsso-to-the-software-catalog.adoc

```diff
@@ -0,0 +1,92 @@
+[id="provisioning-users-from-rhsso-to-the-software-catalog"]
+= Creating a custom transformer to provision users from Red Hat Single-Sign On (RHSSO) to the software catalog
+
+To customize further how users/groups are mapped into entities in {product}, you can create a custom backend module that utilizes the `keycloakTransformerExtensionPoint` to provide custom user and group transformers to the keycloak backend.
+
+.Prerequisites
+* You xref:provisioning-users-from-rhsso-to-the-software-catalog[enabled provisioning users from Red Hat Single-Sign On (RHSSO) to the software catalog].
+
+.Procedure
+. Create a new backend module with the `yarn new` command.
+
+. Add your custom user and group transformers to the `keycloakTransformerExtensionPoint`.
+
++
+Below is an example of how the backend module can be defined:
++
+.`plugins/__<module-name>__/src/module.ts`
+[source,javascript]
+----
+/* highlight-add-start */
```

Comment:
remove `/* highlight-add-start */` and `/* highlight-add-end */` since it doesn't seem to render properly in the docs

Reply:
```suggestion
```